src/components/ListItem.jsx

```diff
@@ -15,6 +15,27 @@ export function ListItem({ item, listPath }) {
 		isLessThan24HoursSinceLastPurchased,
 	);
 
+	const [urgency, setUrgency] = useState('');
```

Comment:
hey @flbarfield @aberger3647 , you can probably just set urgency as a const since it should re render every time an item is checked off: 
```
	const urgency = determineUrgency(
		dateLastPurchased?.toDate(),
		dateNextPurchased?.toDate(),
	);

```

and instead of setting state in `determineUrgency` you can just return the strings instead. and then you also won't need the useEffect